### PR TITLE
Improve MSTest SDK doc

### PR DESCRIPTION
**Pull request by Evangelink**
## Summary

Make it clear that the global.json is the recommended way and add links to `MSTest runner` and `VSTest` docs.

Relates to https://github.com/microsoft/testfx/issues/2703

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/testing/unit-testing-mstest-sdk.md](https://github.com/dotnet/docs/blob/04b75ddf81cad6a1ae6e065bf58587580a5e2d52/docs/core/testing/unit-testing-mstest-sdk.md) | [MSTest SDK overview](https://review.learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-mstest-sdk?branch=pr-en-us-40725) |
| [docs/csharp/linq/standard-query-operators/join-operations.md](https://github.com/dotnet/docs/blob/04b75ddf81cad6a1ae6e065bf58587580a5e2d52/docs/csharp/linq/standard-query-operators/join-operations.md) | [Join Operations in LINQ](https://review.learn.microsoft.com/en-us/dotnet/csharp/linq/standard-query-operators/join-operations?branch=pr-en-us-40725) |
| [docs/standard/net-standard.md](https://github.com/dotnet/docs/blob/04b75ddf81cad6a1ae6e065bf58587580a5e2d52/docs/standard/net-standard.md) | [.NET Standard](https://review.learn.microsoft.com/en-us/dotnet/standard/net-standard?branch=pr-en-us-40725) |
| [docs/whats-new/dotnet-docs-mod1.md](https://github.com/dotnet/docs/blob/04b75ddf81cad6a1ae6e065bf58587580a5e2d52/docs/whats-new/dotnet-docs-mod1.md) | [".NET docs: What's new for April 2024"](https://review.learn.microsoft.com/en-us/dotnet/whats-new/dotnet-docs-mod1?branch=pr-en-us-40725) |

### CODE REVIEW

**Review by MarcoRossignoli — APPROVED** [review]

**Review by IEvangelist — APPROVED** [review]

File: docs/core/testing/unit-testing-mstest-sdk.md [hunk 1]
```diff
@@ -33,15 +33,40 @@ How to use the `MSTest.Sdk` in a project:
...
+
+```json
+{
+    "msbuild-sdks": {
+        "MSTest.Sdk": "3.3.1"
+    }
+}
+```
+
+For more information, see [Use MSBuild project SDKs](/visualstudio/msbuild/how-to-use-project-sdk?#how-project-sdks-are-resolved).
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-sdk.md:61]:
```suggestion
For more information, see [Use MSBuild project SDKs](/visualstudio/msbuild/how-to-use-project-sdk#how-project-sdks-are-resolved).
```

File: docs/core/testing/unit-testing-mstest-sdk.md [hunk 2]
```diff
@@ -33,15 +33,40 @@ How to use the `MSTest.Sdk` in a project:
...
+For more information, see [Use MSBuild project SDKs](/visualstudio/msbuild/how-to-use-project-sdk#how-project-sdks-are-resolved).
 
 When you `build` the project, all the needed components are restored and installed using the standard NuGet workflow set by your project.
 
 You don't need anything else to build and run your tests and you can use the same tooling (for example, `dotnet test` or Visual Studio) used by a ["classic" MSTest project](./unit-testing-with-mstest.md).
 
 ## Select the runner
 
-By default, MSTest SDK relies on MSTest runner, but you can easily switch to VSTest by adding the property `<UseVSTest>true</UseVSTest>`.
+By default, MSTest SDK relies on [MSTest runner](./unit-testing-mstest-runner-intro.md), but you can easily switch to [VSTest](/visualstudio/test/vstest-console-options.md) by adding the property `<UseVSTest>true</UseVSTest>`.
```
Comment (IEvangelist) [inline docs/core/testing/unit-testing-mstest-sdk.md:69]:
```suggestion
By default, MSTest SDK relies on [MSTest runner](./unit-testing-mstest-runner-intro.md), but you can easily switch to [VSTest](/visualstudio/test/vstest-console-options) by adding the property `<UseVSTest>true</UseVSTest>`.
```

**Comment by Evangelink** [comment]
Thanks for the review @IEvangelist. The error is unrelated to this PR, could you force merge?

**Comment by IEvangelist** [comment]
> Thanks for the review @IEvangelist. The error is unrelated to this PR, could you force merge?

I fixed the issue in `main`. Would you be willing/able to rebase, to pick up that fix?

**Comment by Evangelink** [comment]
Done

**Pull request merged**